payroll-processor-functions/src/payroll-processor-functions/Features/Payrolls/PayrollTrigger.cs

```diff
@@ -0,0 +1,156 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage.Queue;
+using Microsoft.WindowsAzure.Storage.Table;
+using PayrollProcessor.Functions.Features.Employees;
+using PayrollProcessor.Functions.Infrastructure;
+using System;
+using System.Threading.Tasks;
+
+namespace PayrollProcessor.Functions.Features.Payrolls
+{
+    public class PayrollTrigger
+    {
+        [FunctionName("Payroll_GetAll")]
```

Comment:
I should probably use the `nameof()` operator here instead of custom naming the functions different than the method names.

Reply:
Updated to use `nameof()`